src/InlineLogo/InlineLogo.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import InlineLogo from './InlineLogo';
+import renderer from 'react-test-renderer';
+
+it('renders without crashing', () => {
+	const div = document.createElement('div');
+	ReactDOM.render(
+		<InlineLogo>
+			<div>test</div>
+		</InlineLogo>,
+		div
+	);
+	ReactDOM.unmountComponentAtNode(div);
+});
+
+it('replaces Codestar in a string with an image', () => {
+	const comp = renderer.create(
+		<InlineLogo>
+			<div>test Codestar test</div>
+		</InlineLogo>
+	);
+	const compInstance = comp.getInstance();
+	const domResult = compInstance.render()[0].props.children;
+	expect(domResult[0]).toBe('test ');
+	expect(typeof domResult[1].type).toBe('function');
```

Comment:
It took me some time to figure out why `domResult[1].type` should be a function and that this `Codestar` text should be replaced with some image. Can you add a comment about this above this test?